docs/cli-usage.rst

```diff
@@ -170,6 +170,25 @@ In `/path/to/helper.sh`:
 
     #!/bin/bash
     pass show path/to/password | head -n 1
+    
+Example for an `environnement variable`:
+
+.. code-block:: ini
+
+   [global]
+   default = somewhere
+   ssl_verify = true
+   timeout = 5
+
+   [somewhere]
+   url = http://somewhe.re
+   private_token = helper: printf -- '%%s' ${GITLAB_TOKEN}
+   timeout = 1
+
+**Notice:**
+
+   * ``printf`` is the executable provided by coreutils package (on a debian based distribution).
+   * ``%`` must be escaped using an other `%` (in order to allow `configparser` load configuration properly.
```

Comment:
Thank you for this contribution @glennie! Initially I wanted to say this would probably be superseded by https://github.com/python-gitlab/python-gitlab/pull/1296, but I like your idea because you can define multiple env vars (e.g. one per gitlab id entry).

Just one question: Wouldn't simply using `echo -n ${VARIABLE}` achieve the same? Have you tried this? `-n` should strip the newline.